TrackerMC/src/StrawDigisFromStrawGasSteps_module.cc

```diff
@@ -837,7 +833,7 @@ namespace mu2e {
       //  sums voltages from both waveforms for ADC
       ADCVoltages wf[2];
       // add the jitter in the EventWindowMarker time for this Panel (constant for a whole microbunch, same for both sides)
-      double dt = _ewMarkerROCdt[sid.getPanel()];
+      double dt = _ewMarkerROCdt[sid.uniquePanel()];
```

Comment:
For future improvements: is to too late to define some types so that cross-stitched indexing will be caught at the compiler level?

Reply:
Changing panel and uniquepanel to different types would require a lot of new code, extending up into the database tables and the proditions interface.  It is way beyond scope for this PR.